spec/fork_break_spec.rb

```diff
@@ -86,6 +86,20 @@ def counter_after_synced_execution(counter_path, with_lock)
     end
   end
 
+  it 'ignores breakpoints when running outside a ForkBreak process' do
+    class Foo
+      include ForkBreak::Breakpoints
+
+      def bar
+        breakpoints << :test
+        'baz'
+      end
+    end
+
+    expect { Foo.new.bar }.to_not raise_error
+    expect(Foo.new.bar).to eq('baz')
+  end
```

Comment:
Don't think it's possible with the way the break point setter is currently being handled.